source/UberRidesTests/RidesClientTests.swift

```diff
@@ -1083,4 +1087,11 @@ class RidesClientTests: XCTestCase {
         let accessToken = ridesClient.fetchAccessToken()
         XCTAssertNil(accessToken)
     }
+    
+    //MARK: Helper
+    func saveFirstRunKey() {
+        let kCheckFirstRun = "com.uber.checkFirstRun"
```

Comment:
let's make `kCheckFirstRun` internal in TokenManager and use that value here rather than declaring it again